src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -752,7 +752,7 @@ public Task GetClusterCertificates()
 
             var pfx = collection.Export(X509ContentType.Pfx);
 
-            var contentDisposition = "attachment; filename=ClusterCertificatesCollection.pfx";
+            var contentDisposition = "attachment; filename=ServerCertificatesCollection.pfx";
```

Comment:
@gregolsky Can it breaks something in cloud? Do you really on that name?

Reply:
any - it might be risky change IMHO

Reply:
No, we do not rely on it. I'll share more info in a PM to you.